controllers/useraccount/useraccount_controller.go

```diff
@@ -475,6 +475,13 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	logger := log.FromContext(ctx)
 	logger.Info("deleting the User resources")
 
+	//get the UID before deleting the user
+	// TO DO: this is a workaround for breaking change introduced by console with upgrade to 4.15. This remove resources created for users logging in from OIDC which don't have owner references after 4.15
```

Comment:
```suggestion
	// TO DO: this is a workaround for breaking change introduced by console with upgrade to 4.15. This remove resources created for users logging in from OIDC which don't have owner references starting from OCP 4.15 version.
```

Reply:
fixed in [a224e71](https://github.com/codeready-toolchain/member-operator/pull/568/commits/a224e71fffd02f8735473c20088feb02a2eaa6b0)